tests/puppeteer/campaigns/smoke/crawlingBO.js

```diff
@@ -0,0 +1,65 @@
+// importing pages
+const LOGIN_PAGE = require('../../pages/BO/login');
+const DASHBOARD_PAGE = require('../../pages/BO/dashboard');
+const COMMON_PAGE = require('../../pages/BO/commonPage');
+const ORDERS_PAGE = require('../../pages/BO/orders');
+const PRODUCTS_PAGE = require('../../pages/BO/products');
+const CUSTOMERS_PAGE = require('../../pages/BO/customers');
+
+let page;
+let LOGIN;
```

Comment:
```suggestion
let loginPage;
```
This variable holds the instance of the class, therefore it needs to be in this format: `loginPage` (CamelCase).